Content.Shared/CCVar/CCVars.cs

```diff
@@ -369,6 +369,12 @@ public sealed class CCVars : CVars
         public static readonly CVarDef<string> SecretWeightPrototype =
             CVarDef.Create("game.secret_weight_prototype", "Secret", CVar.SERVERONLY);
 
+        /// <summary>
+        /// Role id for the Discord webhook to ping when the round ends.
+        /// </summary>
```

Comment:
copy pasted comment

Reply:
Fixed in [`d8b9797` (#25741)](https://github.com/space-wizards/space-station-14/pull/25741/commits/d8b9797ab70713844dde1bc21c044e14af659a4e)